code/modules/mob/new_player/new_player.dm

```diff
@@ -261,6 +261,7 @@
 	if(!is_job_whitelisted(src, rank))	 return 0
 	if(!job.player_old_enough(client))	return 0
 	if(job.admin_only && !(check_rights(R_EVENT, 0))) return 0
+	if(job.mentor_only && !(check_rights(R_ADMIN|R_MOD|R_MENTOR, 0))) return 0
```

Comment:
The R_MOD right is a carryover from baystation and (afaik) has never been used, nor is it planned to be. Don’t know if this is necessary or not.